src/main/java/webserver/response/ResponseHandler.java

```diff
@@ -0,0 +1,56 @@
+package webserver.response;
+
+import db.Database;
+import model.User;
+import webserver.loader.FileResult;
+import webserver.util.ContentType;
+import webserver.http.HttpRequest;
+import webserver.loader.FileResourceLoader;
+import webserver.loader.ResourceLoader;
+import webserver.request.RequestParser;
+
+import java.io.DataOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+
+public class ResponseHandler {
+
+    private final DataOutputStream dos;
+    private final HttpRequest httpRequest;
+    private final ResponseWriter responseWriter;
+
+    private final String GET = "GET";
+    private final String REGISTRATION_ACTION = "/create";
+    private final String DEFAULT_MAIN_PAGE = "/index.html";
+    private final String ROOT_PATH = "/";
+
+    public ResponseHandler(HttpRequest request, OutputStream out) {
+        this.dos = new DataOutputStream(out);
+        this.responseWriter = new ResponseWriter(dos);
+        this.httpRequest = request;
+    }
+
+    public void dispatch() throws IOException {
+        if (httpRequest.getMETHOD().equals(GET)) {
+            handleGetRequest();
+        }
+    }
+
+    private void handleGetRequest() throws IOException {
+        String urlPath = httpRequest.getURL_PATH();
+        if (httpRequest.getPathWithoutQuery().equals(REGISTRATION_ACTION)) {
+            User justJoinedUser = RequestParser.parseRegistrationData(urlPath);
```

Comment:
```suggestion
            User justJoinedUser = new User(Request.getParam("user"), ...) 
```

Request 객체가 있는데 또 파서를 통해 뭔가 한다는 게 좀 어색해 보입니다.